packages/cli/src/generator/copyProjectTemplateAndReplace.js

```diff
@@ -17,7 +17,7 @@ const walk = require('../util/walk');
  * Util for creating a new React Native project.
  * Copy the project from a template and use the correct project name in
  * all files.
- * @param srcPath e.g. '/Users/martin/AwesomeApp/node_modules/react-native/local-cli/templates/HelloWorld'
+ * @param srcPath e.g. '/Users/martin/AwesomeApp/node_modules/react-native/templates/HelloWorld'
```

Comment:
chore: `react-native/template` instead of `templates/HelloWorld`.

Reply:
fixd